examples/tfengine/team.hcl

```diff
@@ -75,20 +58,62 @@ template "devops" {
   }
 }
 
+template "cicd" {
+  recipe_path = "{{$recipes}}/cicd.hcl"
+  output_path = "./cicd"
+  data = {
+    project_id = "example-devops"
+    github = {
+      owner = "GoogleCloudPlatform"
+      name  = "example"
+    }
+    branch_regex   = "^master$"
+    terraform_root = "terraform"
+
+    # Prepare and enable default triggers.
+    triggers = {
+      validate = {}
+      plan     = {}
+      apply    = { run_on_push = false } # Do not auto run on push to branch
+    }
+
+    build_viewers = [
+      "group:example-cicd-viewers@example.com",
+    ]
+
+    # Kubernetes intentionally left out as it cannot be deployed by CICD.
+    managed_modules = [
+      "bootstrap", // NOTE: CICD service account can only update APIs on the devops project.
```

Comment:
did you try adding/removing a new API and let CICD deploys the change?

Reply:
Verified, works fine